### ISSUE TITLE
[Feature request] Custom Colors for Counters

### ISSUE THREAD

**Issue by a-tokarev** [albertvaka/bettercounter#63]
Please add the ability to pick custom colors for counters.
Not for aesthetics as such, but for usability! Sometimes the color of a counter means something to you and will help to visually assosiate the counter with the objects being counted  (e.g., clothing items, flowers, fruit etc.) and to find the corresponding counter faster on the screen.

**Comment by voidplayer** [comment]
I dont even understand how current colors work

If you let default colors it picks a random color you you cant chose from the list

**Comment by albertvaka** [comment]
We changed the available colors in the list at some point, so if you had an older counter it might have a color that is no longer available for new counters but can still be used on existing counters. It's not "a random color", unless there's a bug.

**Comment by voidplayer** [comment]
Oh, I see thanks!

**Comment by a-tokarev** [comment]
@albertvaka, could you please add the color picker and let users choose custom colors for their counters that they want (instead of 8 preset colors)?

**Comment by albertvaka** [comment]
I don't think this is a big enough improvement to justify the work that I would have to put into it, but let's keep the issue open to see how many users would want this.

**Comment by voidplayer** [comment]
Current palette is rather bland, it feels like different shades of puke. Why was the old palette changed again?

More distinct colors would be nice, but i only have 9-10 counters and i am using the color for the category, so i manage with just 4 of them

Good enough :)

**Comment by a-tokarev** [comment]
> to see how many users would want this

Chances are, nobody else would want this in years, because some little features need to be implemented first before their value could be appreciated. BETTER counter, eh? :)

**Comment by voidplayer** [comment]
> The reason to change it was #56

I see, readability can be an issue indeed i dont know much about color theory to make it readable on all situations

I liked all palette best because colors were more different. Right now theres 4 very similar blues, 2 purples, 3 orange/brown/green and black

Good enough for me because i only have 4 categories :)

**Comment by impeira** [comment]
The 9 preset colors seem a bit limited for my taste.

I would like to group my counters by activity. E.g. sport activities in slightly different green tones, social events in blue tones, bad habits in grey tones and so on.

**Comment by Mrnofish** [comment]
FWIW I would definitely appreciate more color presets and the ability to change them.

Ideally, you'd want to adjust the text color automatically to ensure readability, too, but if given the choice I'd rather have more presets and customizable, but with black text, than no change at all.
